report/renderer/util.js

```diff
@@ -636,6 +636,11 @@ Util.UIStrings = {
 
   /** Descriptive explanation for environment throttling that was provided by the runtime environment instead of provided by Lighthouse throttling. */
   throttlingProvided: 'Provided by environment',
+
+  /** Descriptive explanation used when an audit is not part of the calculated score. */
+  unscoredLabel: 'Unscored',
+  /** Descriptive explanation used when an audit is not part of the calculated score. */
+  unscoredTitle: 'This audit is not part of the category score.',
```

Comment:
my mind went here, but not sure what will be best received by unrelated parties

```suggestion
  unscoredTitle: 'This audit does not contribute to the overall category score.',
```